applications/nrf5340_audio/README.rst

```diff
@@ -820,7 +828,14 @@ Use this cable to connect the audio source (PC) to the analog **LINE IN** on the
 Configuring FOTA upgrades
 =========================
 
-You can configure FOTA upgrades to replace the applications on both the application core and the network core.
+.. important::
```

Comment:
```suggestion
.. note::
```

Shouldn't this be just a note?

Reply:
This is very important information, so I think it should be important::

Reply:
Then use caution. We usually use only 3 kinds of notes: note, caution and warning (for risk of damage or injury)

Reply:
Changed to caution